### ISSUE TITLE
Need to force full sync every time I launch HDT

### ISSUE THREAD

**Issue by Kaighal** [Epix37/Hearthstone-Deck-Tracker#2257]
Hello,

As stated in the title, I need to force full sync with Heartstats every time I launch HDT, because otherwise, I am stuck with older versions of my decks.

I did not always have this issue, I think it appeared when I updated one of my decks on a different computer than the one I created it with (I have 2 computers at home and play HS on both). Everything worked fine on the 2 computers before that day.

Since then, every deck I update remains stuck at its old version. New decks are imported properly (only once, I think) but aren't updated as well if I make any change to them.

I'll give you an example to make things clear : the deck that I initially had this issue with is called "Highlander Warrior", it's a Reno deck and it was at version 1.9.

I changed it and saved it as v1.10 on my 2nd computer. But I get v1.9 every time I launch HDT. So, I tried to change it again on my 1st computer (v1.11), I tried to save it without any change (v1.12) and I still get v1.9 when I launch the software.

It happened about a month ago (approximately), the software version is up to date and updates didn't solve this issue.

Can you help me please ? Thanks :)

**Comment by Epix37** [comment]
I think this might be the same problem: #2186, please see if you find a solution there.

**Comment by Kaighal** [comment]
Oh, I should have turned a few pages before posting.

Auto Delete was already off so I just deleted all the decks then force-full-sync'ed as indicated, got my most-recent decks back. I closed HDT, ran it again and they were still there (and up to date). The problem appears to be solved :). Many thanks !

Sorry to have bothered you for an already-known issue. Have you found out what was its cause, can I help in any way about that ?
